tests/src/test/scala/cats/tests/EitherTests.scala

```diff
@@ -10,6 +10,7 @@ class EitherTests extends CatsSuite {
   implicit val iso = CartesianTests.Isomorphisms.invariant[Either[Int, ?]]
 
   checkAll("Either[String, Int]", GroupLaws[Either[String, Int]].monoid)
+  checkAll("Semigroup[Either[String, Int]]", SerializableTests.serializable(Semigroup[Either[String, Int]]))
```

Comment:
Maybe check `Monoid[Either[String, Int]]` here?
The instance tested should be the same, but might be clearer given that we are testing `GroupLaws.monoid` above?